x-pack/legacy/plugins/siem/public/pages/home/index.tsx

```diff
@@ -142,8 +142,18 @@ export const HomePage = pure(() => (
                   <Switch>
                     <Redirect from="/" exact={true} to="/overview" />
                     <Route path="/:pageName(overview)" render={() => <Overview />} />
-                    <Route path="/:pageName(hosts)" component={HostsContainer} />
-                    <Route path="/:pageName(network)" component={NetworkContainer} />
+                    <Route
+                      path="/:pageName(hosts)"
+                      render={({ match, location }) => (
+                        <HostsContainer url={match.url} location={location} />
+                      )}
+                    />
+                    <Route
+                      path="/:pageName(network)"
+                      render={({ match, location }) => (
+                        <NetworkContainer url={match.url} location={location} />
+                      )}
+                    />
```

Comment:
Note: Both of these MUST be `render` in order to avoid passing down the `match` as it is a newly created object every single time Route decides to render (which can be a lot).

Also even though the downstream React.memo component doesn't need `location`, adding the `location` causes it to update correctly and re-render when we switch tabs (which is what we want)